game_start/game.js

```diff
@@ -32,66 +32,64 @@ const result = 0;
 // logika gry
 
 const showCards = function () {
-	activeCard = this;
-	activeCard.classList.remove('hidden');
-
-	if (activeCards.length === 0) {
-		activeCards[0] = activeCard;
-		return;
-	} else {
-		cards.forEach((card) => card.removeEventListener('click', showCards));
-		activeCards[1] = activeCard;
-
-		setTimeout(function () {
-			if (activeCards[0].className === activeCards[1].className) {
-				activeCards.forEach((card) => {
-					return;
-				});
-			} else {
-				activeCards.forEach((card) => {
-					card.classList.add('hidden');
-				});
-			}
-			activeCard = '';
-			activeCards.length = 0;
-			cards.forEach((card) => card.addEventListener('click', showCards));
-		}, 1000);
-	}
+  activeCard = this;
+  activeCard.classList.remove('hidden');
+
+  if (activeCards.length === 0) {
+    activeCards[0] = activeCard;
+
+    return;
+  } else {
+    cards.forEach((card) => card.removeEventListener('click', showCards));
+    activeCards[1] = activeCard;
+
+    if (activeCards[0].className !== activeCards[1].className) {
+      setTimeout(function () {
+        activeCards.forEach((card) => {
+          card.classList.add('hidden');
+        });
+
+        activeCard = '';
+        activeCards.length = 0;
+
+        cards.forEach((card) => card.addEventListener('click', showCards));
+      }, 1000);
+    }
+  }
 };
 // licznik kliknięć myszką
 
 const mouseMove = () => {
-	let clicks = 0;
-	console.log(clicks);
+  let clicks = 0;
 
-	const click = () => {
-		clicks++;
-		moves.textContent = `Moves: ${clicks}`;
-	};
+  const click = () => {
+    clicks++;
+    moves.textContent = `Moves: ${clicks}`;
+  };
 
-	return click;
+  return click;
 };
 
 const clicksScore = mouseMove();
 
 // główna funckja
 
 const playGame = () => {
-	cards.forEach((card) => {
-		card.addEventListener('click', clicksScore);
+  cards.forEach((card) => {
+    card.addEventListener('click', clicksScore);
 
-		const position = Math.floor(Math.random() * cardsArray.length);
+    const position = Math.floor(Math.random() * cardsArray.length);
 
-		card.classList.add(cardsArray[position]);
-		cardsArray.splice(position, 1);
-	});
+    card.classList.add(cardsArray[position], 'card-general');
```

Comment:
tutaj możesz podawać nazwy klas po przecinku, więc jak dodasz card-general tutaj to będziesz miał dużo czytelniejszy .css